lma-websocket-stack/deployment/lca-websocket.yaml

```diff
@@ -1482,6 +1488,7 @@ Resources:
             settingsJSON = json.loads(settings)
             settingsJSON['WSEndpoint'] = props['WSEndpoint']
             settingsJSON['recordingDisclaimer'] = props['recordingDisclaimer']
+            settingsJSON['AssistantWakePhraseRegEx'] = props['assistantWakePhraseRegEx']
```

Comment:
We don't actually use the AssistantWakePhraseRegEx here though, do we? I think we could remove this line since the only important aspect is that it is part of the custom resource properties.

Reply:
Agreed. Removed this line. 
(No difference in the outcome either way since this lambda reads the values updated by AI stack before replacing the JSON).